exploration/percent-format.md

```diff
@@ -58,15 +58,47 @@ use one of the existing number-formatting functions `:number` and `:integer` wit
 or use the proposed _optional_ function `:unit` with an option `unit=percent`.
 Combinations of these approached might also be used.
 
+### Unit Scaling
+
+There is a difference between _input_ scaling and _output_ scaling in `MeasureFormat`,
+which is the model for the `:unit` function in Unicode MessageFormat.
```

Comment:
This is not the whole story, though, as almost all of the currently proposed `:unit` options are based on the JS `Intl.NumberFormat` options.